heka/sandbox/filters/telemetry_decoder_view.lua

```diff
@@ -0,0 +1,106 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+Creates a summary view of the TelemetryDecoder Statistics.
+--]]
+
+require "bloom_filter"
+require "circular_buffer"
+require "cjson"
+require "string"
+
+bf                  = bloom_filter.new(3*1e6, 0.01) -- todo up the size when not under the sandbox manager
```

Comment:
What does the sandbox manager have to do with the bloom filter size? Are sbmgr-managed sandboxes limited in the amount of memory they can use?